src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -269,48 +264,23 @@ const formConfig = {
               items: {
                 'ui:title': disabilityNameTitle,
                 'ui:description': facilityDescription,
-                treatments: {
+                vaTreatments: {
                   'ui:options': {
                     itemName: 'Facility',
                     viewField: treatmentView
                   },
                   items: {
-                    // Hopefully we can get rid of this annoying nesting
-                    treatment: {
-                      treatmentCenterName: {
-                        // May have to change this to 'Name of [first]...'
-                        'ui:title': 'Name of VA medical facility'
-                      },
-                      startTreatment: {
-                        'ui:widget': 'date',
-                        'ui:title': 'Approximate date of first treatment since you received your rating'
-                      },
-                      endTreatment: {
-                        'ui:widget': 'date',
-                        'ui:title': 'Approximate date of last treatment'
-                      }
-                      // I think we're planning on filling these in with the typeahead?
-                      // treatmentCenterType: {
-                      //   'ui:options': {
-                      //     hideIf: () => true
-                      //   }
-                      // },
-                      // treatmentCenterCountry: {
-                      //   'ui:options': {
-                      //     hideIf: () => true
-                      //   }
-                      // },
-                      // treatmentCenterState: {
-                      //   'ui:options': {
-                      //     hideIf: () => true
-                      //   }
-                      // },
-                      // treatmentCenterCity: {
-                      //   'ui:options': {
-                      //     hideIf: () => true
-                      //   }
-                      // }
-                    }
+                    treatmentCenterName: {
+                      'ui:title': 'Name of VA medical facility'
+                    },
+                    treatmentDateRange: dateRangeUI(
+                      'Approximate date of first treatment',
+                      'Approximate date of last treatment',
+                      'Date of last treatment must be after date of first treatment'
+                    ),
+                    // TODO: Put these back as hidden in the UI once typeahead fills this out
```

Comment:
It's probably find to do this before the typeahead fills these fields out, but I'd say don't worry about it for now. Can you write a ticket to make sure this happens later?